src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -133,7 +133,9 @@ class CourtService(
       )
     }
 
-    trackVideoLinkBookingUpdated(booking)
+    videoLinkBookingRepository.flush()
```

Comment:
I am surprised this is needed, given that it is all done in a single transaction.

Reply:
It needs an Id before the application insights listener fires.  That probably won't happen without the flush().

Reply:
Ah right... but didn't it work before?

Reply:
No.  Ids were null in the App insights log. Just hadn't got round to fixing it.